frontend/app/(footer)/event/create/page.tsx

```diff
@@ -17,7 +17,10 @@ import { useRouter } from "next/navigation";
 import { FormEvent, useState } from "react";
 
 export type FormData = {
-  date: string;
+  end: string;
```

Comment:
what is the point of this new `start` and `end` variables?

Reply:
its for the event summary just before the event is created - too clunky having so many dates and times for end and start so I wanted to combine it but it wouldnt let me format it in that page so i just passed up the info

Reply:
ive removed this now